src/main/java/anichan/parser/CommandParser.java

```diff
@@ -25,7 +25,7 @@
     protected static final String INTEGER_VALUE_OUTSIDE_OF_INTEGER_RANGE = "Please ensure the integer is not larger"
                                                                             + " than " + Integer.MAX_VALUE + ".";
     protected static final String DESCRIPTION_NOT_REQUIRED = "Command does not require additional parameters!";
-    protected static final String REGEX_ALPHANUMERIC = "^[a-zA-Z0-9\\s]*$";
+    public static final String REGEX_ALPHANUMERIC_WITH_SPACE = "^[a-zA-Z0-9\\s]*$";
```

Comment:
Isn't it better for this to be protected?

Reply:
I switched this to public so I could access the regex from the human package. Should I have made a duplicate regex there instead?

Reply:
I'm not really sure about this, but I think if that's the case we should have a class specifically for such scenario, but since huge modification at this stage is not recommended, I think might be better to duplicate the regex in case someone decides to use this regex somewhere too and make modification to it.

Reply:
Alright I have separated them!